### PR TITLE
Exec copy sept 3 2021

### PR DESCRIPTION
**Pull request by prose11**
Not sure if more is needed for DssVest.

More granularity on the PE vesting might be needed?

### CODE REVIEW

File: governance/votes/Executive vote - September 3, 2021.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- 2 million DAI will be distributed to the **[DAI Foundation (DAIF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP17) Reserves Backup Wallet** at **0x5F5c328732c9E52DfCb81067b8bA56459b33921f** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP24).
+- 138,591 DAI will be distributed to the **[DAI Foundation (DAIF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP17) Operational Wallet** at **0x34D8d61050Ef9D2B48Ab00e6dc8A8CA6581c5d63** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP24).
+
+### DssVest Implementation and Initial Streams
+
+As per this [ratification poll](https://vote.makerdao.com/polling/QmPfjETk?network=mainnet#poll-detail), DssVest will be implemented and the following token streams established if this executive proposal passed. For more information on DssVest, please review [MIP54](https://mips.makerdao.com/mips/details/MIP54) and its [forum thread](https://forum.makerdao.com/t/mip-54-dssvest/8025).
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - September 3, 2021.md:56]: if this executive proposal **passed** -> if this executive proposal **passes**

File: governance/votes/Executive vote - September 3, 2021.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+**DAI Streams**
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - September 3, 2021.md:58]: You should list the starting date as well?

File: governance/votes/Executive vote - September 3, 2021.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+**MKR Streams**
+
+1,051.25 MKR will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** through June 30, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15).
+
+803.1833 MKR will be streamed to the **[Growth (GRO-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4)** through July 1, 2022 at **0x7800C137A645c07132886539217ce192b9F0528e** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP19).
+
+**No more than** 15,422.5 MKR will be streamed to team members of the **[Protocol Engineering (PE-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7)** through April 30, 2025, at individual addresses maintained by the [Core Unit Facilitator](https://mips.makerdao.com/mips/details/MIP41c4SP7) according to the active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7).
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - September 3, 2021.md:84]: I don't think we can say 'no more than' in here. We need to explain exactly what voting for this does, not leave it ambiguous. Like this proposal passing distributes x MKR, right? If they want to add additional recipients, that'll need to happen in a future executive vote.

File: governance/votes/Executive vote - September 3, 2021.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+300,050 DAI will be streamed to the **[Growth (GRO-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4)** through October 31, 2021 at **0x7800C137A645c07132886539217ce192b9F0528e** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP19).
+
+4,080,000 DAI will be streamed to the **[Protocol Engineering (PE-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7)** through April 30, 2022 at **0xe2c16c308b843eD02B09156388Cb240cEd58C01c** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7).
+
+**MKR Streams**
+
+1,051.25 MKR will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** through June 30, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15).
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - September 3, 2021.md:80]: The dates are correct here?

Comment (LongForWisdom) [reply governance/votes/Executive vote - September 3, 2021.md:80]: I figured some of these would be 4 year as well.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Would like to see everything confirmed on the main spreadsheet as well.

File: governance/votes/Executive vote - September 3, 2021.md (hunk 3 above)
Comment (prose11) [reply governance/votes/Executive vote - September 3, 2021.md]: This should cover it, and it plainly visible in the code so we're not adding anything that could further unveil them

File: governance/votes/Executive vote - September 3, 2021.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+**MKR Streams**
+
+1,051.25 MKR will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** starting July 1, 2021 through July 1, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15). A 1-year cliff is in place, making MKR claimable on July 1, 2022.
+
+803.18 MKR will be streamed to the **[Growth (GRO-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4)** starting July 1, 2021 through July 1, 2022 at **0x7800C137A645c07132886539217ce192b9F0528e** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP19). A 1-year cliff is in place, making MKR claimable on July 1, 2022.
+
+13,930 MKR will be streamed to 14 team members of the **[Protocol Engineering (PE-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7)** as detailed below, in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7). All team members have a 1-year cliff and will have their 995 MKR streamed over 4 years.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - September 3, 2021.md:86]: Maybe '13930 MKR total' for the first sentence.

**Review by LongForWisdom — APPROVED** [review]
Suggest minor wording change, but generally this is fine.

**Pull request merged**